tests/test_DG_advection.py

```diff
@@ -13,9 +13,7 @@ def setup_DGadvection(dirname, vector=False):
 
     mesh = IcosahedralSphereMesh(radius=R,
                                  refinement_level=refinements)
-    global_normal = Expression(("x[0]/sqrt(x[0]*x[0]+x[1]*x[1]+x[2]*x[2])",
-                                "x[1]/sqrt(x[0]*x[0]+x[1]*x[1]+x[2]*x[2])",
-                                "x[2]/sqrt(x[0]*x[0]+x[1]*x[1]+x[2]*x[2])"))
+    global_normal = Expression(("x[0]", "x[1]", "x[2]"))
```

Comment:
It's not just when R is one.  You *never* have to provide a normalised vector here.  We just use it for the direction.